dspace-api/src/main/java/org/dspace/eperson/EPersonCLITool.java

```diff
@@ -361,6 +376,21 @@ private static int cmdModify(Context context, String[] argv) throws AuthorizeExc
                 eperson.setNetid(command.getOptionValue(OPT_NEW_NETID.getOpt()));
                 modified = true;
             }
+            if (command.hasOption(OPT_NEW_PASSWORD.getOpt())) {
+                char[] password1 = consoleService.readPassword("Enter new password for user %s", userName);
+                char[] password2 = consoleService.readPassword("Enter new password again to verify");
```

Comment:
Tiny request.  Could we include a colon+space (": ") or something after the description in password1 and password2?  It looks a little odd on the CLI for the cursor to be at the end of the string, with no spacing in between.  So, something like:
```
char[] password1 = consoleService.readPassword("Enter new password for user %s: ", userName);
char[] password2 = consoleService.readPassword("Enter new password again to verify: ");
```

This is just a tiny usability improvement so it no longer looks like your password may include your username :)

Reply:
I see what you mean.  Fixed.